uberfire-extensions/uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/dropdown/items/LiveSearchSelectorDropDownItemViewImpl.java

```diff
@@ -97,12 +100,23 @@ public void setMultiSelect(boolean multiSelect) {
         this.multiSelect = multiSelect;
     }
 
-    @EventHandler("itemAnchor")
-    public void onItemClick(ClickEvent event) {
+    private void onItemSelected(DomEvent event) {
         presenter.onItemClick();
 
-        if(multiSelect) {
+        if (multiSelect) {
             event.stopPropagation();
         }
     }
+
+    @EventHandler("itemAnchor")
+    public void onItemClick(ClickEvent event) {
+        onItemSelected(event);
+    }
+
+    @EventHandler("itemAnchor")
+    void onEnterKeyDown(KeyDownEvent event) {
+        if (event.getNativeKeyCode() == 13) {
```

Comment:
Could we please use https://github.com/gwtproject/gwt/blob/master/user/src/com/google/gwt/event/dom/client/KeyCodes.java#L271 ?

Reply:
Thanks @jomarko for the observation. I'll add the change.